calpip/Task1/Task 1-1.txt

```diff
@@ -1,21 +1,21 @@
 Dear Sir or Madam,
 
-My name is Wellington and I work voluntarily in a community organization that 
-makes a wonderful job for local children. I am writing to ask your company for
+My name is Wellington and I volunteer for a community organization that 
+provides a wonderful service for local children. I am writing to ask your company for
  a donation, which will be used to help us to organize an event next month.
 
-Although you probably have heard about us before, I would like to explain a 
-little bit about our charity. The organization which I help is called Angels 
-of the Music because we teach poor kids how to play a variety of musical 
+Although you have probably heard about us before, I would like to explain a 
+little bit about our charity. The organization I volunteer for is called Angels 
+of the Music. We teach kids from low income families how to play a variety of musical 
 instruments at no cost. We offer music classes every day in three different 
-parks of the city.
+parks of our city.
 
-Next month we are planning to realize a concert in which all the kids we have 
-taught could play on the main stage in downtown area. It will be a great night 
-full of good music, food and specially children’s happiness.
+Next month, we are planning to host a concert in which all the kids we have 
+taught to play can perform on the downtown area's main stage. We plan for it to be a great night 
+full of good music, food and especially children’s happiness.
 
-However, we don’t have enough instruments for all kids play at once. For that 
-reason, we need your donation to buy extra guitars, bases, etc. in order to 
+However, we don’t have enough instruments for all of the kids to play at once. For that 
+reason, we kindly ask your donation to buy extra guitars, bases, and more, in order to 
```

Comment:
`We need your donation` sounds like you are giving them no choice. I'm not doubting you **need** the instruments, but if you're going to ask someone for something, then it is better to think of words that are the equivalent of `please`. 

`etc.` is used when listing a list you unsure of the end to, or feel too lazy to enumerate completely. As you have no intention to express either of these meanings to your potential donator, it is better to use `and more`, which better implies that you know exactly what you need.